proxy/src/control/pb.rs

```diff
@@ -205,3 +190,37 @@ impl<'a> TryFrom<&'a Event> for common::TapEvent {
         Ok(tap_ev)
     }
 }
+
+impl ctx::transport::Server {
```

Comment:
I initially thought the impls on types defined in a different module seemed a big confusing, but given that we wouldn't expect to use these methods anywhere outside of `tap`, I think this makes sense in context 👍